reddit.js

```diff
@@ -1,5 +1,42 @@
+//Problem #1 ForEach by Titile
 var reddit = require("./data.json")
+var posts = reddit.data.children;
+
+posts.forEach(function (redChild) {
+	console.log(redChild.data.title);
+}
+
+//Problem #2 Map Iterators with Reddit
+
+var linkMap = posts.map(function (person) {
```

Comment:
Here, ```post``` or ```child``` would be a more appropriate parameter name than ```person```, since what's being iterated over are the individual posts in the ```posts``` array.